src/app/core/draft/draft.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { HttpService } from '../http/http.service';
+import { AppConfig } from '../../app.config';
+import { HttpErrorService } from '../http/http-error.service';
+import { CaseEventData } from '../../shared/domain/case-event-data';
+import { Observable } from '../../../../node_modules/rxjs';
+import { Draft } from '../../shared/domain/draft';
+import { CaseView } from '../cases/case-view.model';
+
+@Injectable()
+export class DraftService {
+  constructor(
+    private http: HttpService,
+    private appConfig: AppConfig,
+    private errorService: HttpErrorService
+  ) {}
+
+  createDraft(jid: string, ctid: string, eventData: CaseEventData): Observable<Draft> {
+    const saveDraftEndpoint = this.appConfig.getCaseDataUrl()
+      + `/caseworkers/:uid/jurisdictions/${jid}/case-types/${ctid}/event-trigger/${eventData.event.id}/drafts/`;
```

Comment:
can we move this url construction logic inside a 
`appConfig.getDraftsUrl(jid, ctid, eventId)`
method?
This way you can reused it below also remove the duplication